app/views/public/api_documentation.html.erb

```diff
@@ -359,6 +360,22 @@ curl -X POST -H "Token: secret-api-token" -H "Content-Type: application/json" -d
 
 <hr />
 
+<h4 id="ping-thing-batch">Ping Thing Post Batch</h4>
+
+<p>The ping thing batch endpoint allows you to push multiple transaction informations at once.</p>
+<p>Request:</p>
+<pre>curl -X POST -H "Token: secret-api-token" -H "Content-Type: application/json" -d '{"transactions": ["time": "NN", "signature": "signature", "transaction_type": "transfer"}]'
```

Comment:
```suggestion
<pre>curl -X POST -H "Token: secret-api-token" -H "Content-Type: application/json" -d '{"transactions": [{ "time": "NN", "signature": "signature", "transaction_type": "transfer" }, { second_example } ]}'
```

`reported_at` is missing. Also it would be nice to add second example since it's a batch. We should stick to ISO 8601 format https://stackoverflow.com/a/15952652/3868595